.travis.yml

```diff
@@ -168,11 +168,13 @@ before_install:
 
 install:
   - source scripts/install-ci-dependencies.sh
+  - export HELICS_DEPENDENCY_FLAGS=""
+  - export HELICS_OPTION_FLAGS="${OPTION_FLAGS}"
 
 script:
   - mkdir build && cd build
-  - export HELICS_DEPENDENCY_FLAGS="-DZeroMQ_INSTALL_PATH=${CI_DEPENDENCY_DIR}/zmq -DBOOST_INSTALL_PATH=${CI_DEPENDENCY_DIR}/boost"
-  - export HELICS_OPTION_FLAGS="-DBUILD_C_SHARED_LIB=ON -DBUILD_HELICS_EXAMPLES=ON -DBUILD_PYTHON_INTERFACE=ON -DBUILD_JAVA_INTERFACE=ON -DEXAMPLES_WARNINGS_AS_ERROR=ON -DPYTHON_LIBRARY=$(python3-config --prefix)/lib/libpython3.6m.${SHARED_LIB_EXT} -DPYTHON_INCLUDE_DIR=$(python3-config --prefix)/include/python3.6m/"
```

Comment:
just to be clear, the -DBUILD_HELICS_EXAMPLES=ON was removed because that is the default setting, correct?

Reply:
Yes. I think most of the flag setup will get moved later to the install section, to simplify builds with a custom script section.